usdf/object-store.rst

```diff
@@ -0,0 +1,86 @@
+#########################
+Data Access: Object Store
+#########################
+
+Some data at the USDF is stored in S3-compatible object stores.
+In most cases, access to them is via the Data Butler and transparent to users.
+This document describes techniques for advanced direct usage of the object stores.
+
+Storage Locations
+=================
+
+All object stores currently use a single S3 endpoint: ``https://s3dfrgw.slac.stanford.edu``.
+This is typically pointed to by the ``S3_ENDPOINT_URL`` environment variable.
+Eventually the production embargo storage is expected to have its own endpoint.
+
+Within that endpoint, there are many available buckets.
+Some of them are listed below.
+
+Embargo storage
+---------------
+
+These buckets hold the raw images from the Summit and the processed data products derived from them (``-users``) during the embargo period.
+There is one set for each test stand in addition to the production set for the Summit.
+
+- rubin-summit
+- rubin-summit-users
+- rubin-bts
+- rubin-bts-users
+- rubin-tts
+- rubin-tts-users
+
+Other object stores
+-------------------
+
+Prompt Processing development uses a pair of buckets to act as its "central store" and raw image storage (simulating the embargo production storage) as well as another bucket to hold test data.
+
+- rubin-pp
+- rubin-pp-users
+- rubin-prompt-processing-test
+
+Alert Production development uses a bucket to hold cutout visualizations of DiaSources for algorithm analysis.
+
+- rubin-ap-cutouts
+
+
+Credentials
+===========
+
+A single set of credentials is used for read-only access to the raw data buckets and for read-write access to the ``-users`` buckets.
+This set can most easily be accessed by logging into the USDF RSP and starting a notebook server.
```

Comment:
I'd put this line at the top: "The default set of credentials for read-only access is created the first time you log in to the USDF RSP and start a notebook server.", and then reword the stuff below a bit.